1_basics.ipynb

```diff
@@ -25,20 +43,57 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# What is HSR?\n",
+    "\n",
+    "HSR (HSR: Human Support Robot) is one of the \"Toyota Partner Robots\" that support human activities, and it will assist independent living at home for people with disabilities.\n",
+    "\n",
+    "In this notebook, you will learn the overview of the robot by using the HSR simulator."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "# Jupyter Notebookの使い方"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# How to use Jupyter Notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "Jupyter NotebookではCellにPythonコマンドを入力し逐次実行していくことができます。\n",
     "\n",
     "例えば以下のコマンドを実行してみましょう。コマンドを実行するにはセルを選択した状態で「Shift+Enter」を押してください。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "In Jupyter Notebook, you can enter Python commands in the Cells and execute them sequentially.\n",
```

Comment:
If we assume that you are referring to Jupyter Notebooks as a general class, then the following would be better:

    "**In Jupyter Notebooks**, you can enter Python commands **in cells** and execute them sequentially.\n",

Note: I also replaced in "the Cells" with "in cells" because the capitalization used in "Cell" should be unnecessary as it is just a common noun (not unless there is some other reason for it ;)).